ducktape/command_line/defaults.py

```diff
@@ -15,7 +15,7 @@
 import os
 
 
-class ConsoleConfig(object):
+class ConsoleDefaults(object):
```

Comment:
Why the renaming? This doesn't look like it'll impact any other projects since they don't use it, but also seems unimportant -- it wasn't made to make room for something that better fit the `ConsoleConfig` name or anything like that.